src/App/App.js

```diff
@@ -189,5 +184,8 @@ export default App;
 
 App.propTypes = {
```

Comment:
Added propTypes I plan on adding to all files and maybe dive into writing tests for them